tcms/core/widgets.py

```diff
@@ -44,7 +44,13 @@ def render(self, name, value, attrs=None, renderer=None):
             className: "fa fa-paperclip",
             title: "Attach File",
         },
-        "|", "preview", "side-by-side", "fullscreen", "|", "guide"]
+        "|", "preview", "side-by-side", "fullscreen", "|", "guide"
+    ],
+    autosave: {
+        enabled: true,
+        uniqueId: "MyUniqueID",
```

Comment:
this makes SimpleMDE re-use the same text on every page until one of them is saved. Use `window.location` to make it per-page.